### DISCUSSION TITLE
New FF update to v.133 broke TST

### DISCUSSION THREAD

**Discussion by bostiq** [piroor/treestyletab#3673] [Q&A]
Hey, I was wondering if this is happening across the board or it's just me.

My customisations are glitching: my side tab panel is flickering making it hard to make operations on any tabs like reordering nesting, moving, and, while I did overlap my address bar with the title bar to save vertical space, now they are separated again. 

I saved my settings and activated sidebar logs, if useful.

**Comment by SHHSSH** [comment]
@bostiq Yes it will be for everyone as Firefox standard release incorporates new changes.
I will help provide you with some css ASAP to fix your issue as best I can.

I think it's something in here but I may have overlaps from previous versions of Firefox so you may have to rummage - 
 #titlebar {
  visibility: collapse;
}

#main-window[inFullscreen] #sidebar-box {
  display:none !important;
  width: 0px !important;
}

#sidebar-header {
	display: none;
}

#PlacesToolbarItems toolbarseparator { opacity: 0 !important; }

/* remove maximum/minimum width restriction of sidebar */
#sidebar-box {
  max-width: none !important;
  min-width: 11px !important;
}

#main-window #TabsToolbar {
    opacity: 0;
    max-height: 0px;
    pointer-events: none;
}

#navigator-toolbox {
  background-color: rgb(236, 233, 216) !important;
  background-image: linear-gradient(to top, rgba(10, 31, 51, 0.35) 2px, rgba(0, 0, 0, 0.05) 2px, transparent 50%) !important;
}

/* Use Normal top and bottom padding for Compact */
#PlacesToolbarItems .bookmark-item {
  padding-top: 11px !important;
  padding-bottom: 1px !important;
}	

#tabbar tab-closebox {
	display: none; 
}

.menubar-text,
#toolbar-menubar .toolbarbutton-text,
.tabbrowser-tab .tab-label {
  color: black !important;
}

/*
   Override hiding of Bookmarks Toolbar in F11 full screen
   when you roll down the tabs/main toolbar 
*/
#navigator-toolbox[inFullscreen="true"] #PersonalToolbar {
  visibility: unset !important;
}

This is for your userChrome.css in root directory \chrome not TST's css.
Sorry I cannot help with the sidebar panel issues you mentioned though I don't think.

Others also discuss similar things over here - https://github.com/piroor/treestyletab/issues/2147

**Reply by bostiq** [reply]
Hey, Thanks for the reply, good to know I'm not the only one.

And Thanks for the code... I think I managed to fix my css.
